patches/fsharp/0004-Revert-some-F-changes-for-new-APIs.patch

```diff
@@ -0,0 +1,70 @@
+From 10a406e1b68e29aa97cdcaddd354c0916b6829df Mon Sep 17 00:00:00 2001
+From: Chris Rummel <crummel@microsoft.com>
+Date: Mon, 29 Jun 2020 18:31:35 +0000
+Subject: [PATCH 4/4] Revert some F# changes for new APIs.
+
+---
```

Comment:
Any more info? A decent number of patches don't have much explanation.

Reply:
Sorry, I don't have more info on this one.  Many were copied from 3.1.  We'll need to address them during patch removal.